docs/tutorials/06-submit-proofs/05-groth16_proof_submission.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Groth16 Proof Submission
+---
+
+In this tutorial we'll go through the process of submitting and verifying a Groth16 proof to the zkVerify chain.
+
+## Requirements
+
+- A Substrate compatible wallet with funds (ACME) connected to the network.
+- NodeJS and npm installed on your system.
+
+## Step 1: Proof generation with snarkJS
+
+Follow this [this snarkJS guide](https://github.com/iden3/snarkjs?tab=readme-ov-file#guide) to generate a Groth16 proof, along with the corresponding verification key, and public inputs. The tutorial works with any Groth16 proof generated with snarkJs, as long as its number of public inputs does not exceed 16. Both bn128 and bls12381 curves are supported. So, if you already happen to have a snarkJS Groth16 proof at hand, then you can skip this step.
```

Comment:
Maybe we can clarify that bn128 is the same curve used by Ethereum (which they call bn254)